sdk/src/types/block/mana/parameters.rs

```diff
@@ -351,30 +351,29 @@ mod test {
         );
     }
 
-    // TODO: Uncomment when iota.go fixes their calculation + test data
-    // #[test]
-    // fn test_potential_mana_lookup_len_delta() {
-    //     assert_eq!(
-    //         params().potential_mana(
-    //             i64::MAX as u64,
-    //             params().first_slot_of(1),
-    //             params().first_slot_of(params().mana_parameters().decay_factors().len() as u32 + 1)
-    //         ),
-    //         Ok(183827294847826527)
-    //     );
-    // }
-
-    // #[test]
-    // fn test_potential_mana_lookup_len_delta_multiple() {
-    //     assert_eq!(
-    //         params().potential_mana(
-    //             i64::MAX as u64,
-    //             params().first_slot_of(1),
-    //             params().first_slot_of(3 * params().mana_parameters().decay_factors().len() as u32 + 1)
-    //         ),
-    //         Ok(410192222442040018)
-    //     );
-    // }
+    #[test]
+    fn test_potential_mana_lookup_len_delta() {
```

Comment:
Can we remove the test_ prefix ? We usually don't do that, I personally find it a bit redundant

Reply:
sure